internal/functions/efgs/api/structs.go

```diff
@@ -16,8 +16,10 @@ type GenericVerServerResponse struct {
 
 //IssueCodeRequest Issue code request to the Verification server
 type IssueCodeRequest struct {
-	Phone    string `json:"phone"`
-	TestType string `json:"testType"`
+	Phone       string `json:"phone"`
+	TestType    string `json:"testType"`
+	SymptomDate string `json:"symptomDate"` // ISO 8601 formatted date, YYYY-MM-DD
+	TestDate    string `json:"testDate"`
```

Comment:
why is this boilerplate here and we are not using the vf server struct / "API"? https://github.com/google/exposure-notifications-verification-server/blob/main/pkg/api/api.go#L168

Reply:
That's a very... ugly and right question :neutral_face: